tests/Console/ConfigurationResolverTest.php

```diff
@@ -961,6 +961,26 @@ public function provideResolveRulesCases()
                     '@A' => true,
                 ),
             ),
+            '@A - @C' => array(
```

Comment:
ah, right, this is current way, issue is when one subtract not a single rule, but a set (that disable rule as well)
```
{ '@Foo': true, } : @Foo' => { 'AA' => true, 'NOOP' => true, }
{ '@Bar': true, } : @Bar => { 'AA' => false, 'NOOP' => true, }
{ '@Bar': false, } : -@Bar => { 'AA' => true, NOOP' => false, } // as direct revert
```

Remember that `,` means `take later`, so:
- `{ 'X' => true, 'X' => true } = { 'X' => true }`
- `{ 'Y' => true, 'Y' => false } = { 'Y' => false }`
- `{ 'Z' => false, 'Z' => true } = { 'Z' => true }`

IMO:
- `{ '@Foo': true, '@Bar': true } => { 'NOOP' => true, }`, as:
`{ '@Foo': true, '@Bar': true } = { 'AA' => true, 'NOOP' => true, 'AA' => false, 'NOOP' => true, } => {'NOOP' => true, }`
- `{ '@Foo': true, '@Bar': false } => { 'AA' => true, }`, as:
`{ '@Foo': true, '@Bar': false } => { '@Foo': true, '-@Bar': true, } => { 'AA' => true, 'NOOP' => true, 'AA' => true, 'NOOP' => false, } => { 'NOOP' => true, }`